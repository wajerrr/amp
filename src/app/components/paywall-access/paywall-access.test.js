import React from 'react';
import renderer from 'react-test-renderer';
import PaywallAccess from './paywall-access';

describe('PaywallAccess', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<PaywallAccess>paid content</PaywallAccess>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
