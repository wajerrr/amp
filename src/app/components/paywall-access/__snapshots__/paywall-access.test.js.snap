// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaywallAccess should match snapshot 1`] = `
Array [
  <div
    amp-access="access"
  >
    paid content
  </div>,
  <div
    amp-access="NOT access AND userGroup = 'anonymous'"
    amp-access-hide="true"
  >
    Please subscribe to read more
    <br />
    <button>
      Subscribe
    </button>
  </div>,
  <div
    amp-access="NOT access AND userGroup != 'anonymous'"
    amp-access-hide="true"
  >
    Please pay to read more
    <br />
    <button>
      Pay
    </button>
  </div>,
]
`;
